ipynb/country-balance-analysis.ipynb

```diff
@@ -0,0 +1,319 @@
+    "                    --p-min-frequency 10\n",
+    "```\n",
+    "\n",
+    "```    \n",
+    "qiime composition add-pseudocount \\\n",
+    "    --i-table \"$otu_table\".filt.qza \\\n",
+    "    --p-pseudocount 1 \\\n",
+    "    --o-composition-table \"$otu_table\".composition\n",
+    "```    \n",
+    "\n",
+    "```\n",
+    "qiime gneiss correlation-clustering \\\n",
+    "    --i-table \"$otu_table\".composition.qza \\\n",
+    "    --o-clustering \"$otu_table\".tree.nwk.qza\n",
+    "```\n",
+    "\n",
+    "```\n",
+    "qiime gneiss ilr-transform \\\n",
+    "    --i-table \"$otu_table\".composition.qza \\\n",
+    "    --i-tree \"$otu_table\".tree.nwk.qza \\\n",
+    "    --o-balances \"$otu_table\".balances\n",
+    "```\n",
+    "\n",
+    "```\n",
+    "qiime gneiss ols-regression \\\n",
+    "    --p-formula $formula \\\n",
+    "    --i-table \"$otu_table\".balances.qza \\\n",
+    "    --i-tree \"$otu_table\".balances.tree.nwk.qza \\\n",
+    "    --m-metadata-file ag_map_with_alpha.txt \\\n",
+    "    --o-visualization ag_regression.filt.2.summary --verbose\n",
+    "```\n",
+    "\n",
+    "Once we have the summaries generated, we can plug them directly into https://view.qiime2.org/\n",
+    "You may note that one balance that stands out is `y2`.  We'll revisit that balance in a bit.\n",
+    "\n",
+    "From now on, we'll analyze the result using the Gneiss Python API."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {
+    "collapsed": true
+   },
+   "outputs": [],
+   "source": [
+    "from gneiss.composition import ilr_transform\n",
+    "import matplotlib.pyplot as plt\n",
+    "import pandas as pd\n",
+    "import numpy as np\n",
+    "from matplotlib import rc\n",
+    "\n",
+    "from biom import load_table\n",
+    "from skbio import TreeNode\n",
+    "import qiime2\n",
+    "import seaborn as sns\n",
+    "%matplotlib inline"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "We'll load all of the files, and parse out the taxonomies from the biom table."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "/Users/mortonjt/miniconda3/envs/bio/lib/python3.5/site-packages/IPython/core/interactiveshell.py:2698: DtypeWarning: Columns (0,66) have mixed types. Specify dtype option on import or set low_memory=False.\n",
+      "  interactivity=interactivity, compiler=compiler, result=result)\n"
+     ]
+    }
+   ],
+   "source": [
+    "mapping = pd.read_table('ag_map_with_alpha.txt', index_col=0)\n",
+    "\n",
+    "table = load_table('deblur_125nt_no_blooms.biom')\n",
+    "balances = qiime2.Artifact.load(\n",
+    "    'deblur_125nt_no_blooms_rare.biom.balances.qza').view(pd.DataFrame)\n",
+    "tree = qiime2.Artifact.load(\n",
+    "    'deblur_125nt_no_blooms_rare.biom.tree.nwk.qza').view(TreeNode)\n",
```

Comment:
We don't have the artifacts on the FTP. We have the raw data (text, biom, .tree)